examples/plot_nonstationary_error.py

```diff
@@ -0,0 +1,205 @@
+"""
+Non-stationarity of elevation measurement errors
+================================================
+
+Digital elevation models have a precision that can vary with terrain and instrument-related variables. However, quantifying
+this precision is complex and non-stationarities, i.e. variability of the measurement error, has rarely been
+accounted for, with only some studies that used arbitrary filtering thresholds on the slope or other variables (see :ref:`intro`).
+
+Quantifying the non-stationarities in elevation measurement errors is essential to use stable terrain as a proxy for
+assessing the precision on other types of terrain (Hugonnet et al., in prep) and allows to standardize the measurement
+errors to reach a stationary variance, an assumption necessary for spatial statistics (see :ref`spatialstats`).
+
+Here, we show an example in which we identify terrain-related non-stationarities for a DEM difference of Longyearbyen glacier.
+We quantify those non-stationarities by `binning <https://en.wikipedia.org/wiki/Data_binning>`_ robustly
+in N-dimension using :func:`xdem.spstats.nd_binning` and applying a N-dimensional interpolation
+:func:`xdem.spstats.interp_nd_binning` to estimate a numerical function of the measurement error and derive the spatial
+distribution of elevation measurement errors of the difference of DEMs.
+
+**Reference**: `Hugonnet et al. (2021) <https://doi.org/10.1038/s41586-021-03436-z>`_, applied to the terrain slope
+and quality of stereo-correlation (Equation 1, Extended Data Fig. 3a).
+"""
+# sphinx_gallery_thumbnail_number = 8
```

Comment:
what does this mean?

Reply:
The comment signifies which of the plots will make the gallery thumbnail. Unfortunately, it has to show up in the rendered code as well!

Reply:
Thanks for the explanation, good to know!

Reply:
> Unfortunately, it has to show up in the rendered code as well!
I was wondering if it could be hidden! Good to know, it's not taking too much space anyway :)